man/mono.1

```diff
@@ -348,6 +403,12 @@ will switch to '/bin/mono64' iff '/bin/mono' is a 32-bit build).
 \fB--help\fR, \fB-h\fR
 Displays usage instructions.
 .TP
+\fB--hybrid-aot\fR
+This is an experimental flag that instructs the Mono runtime to not
+generate any IL at runtime and depend exclusively on the code
```

Comment:
I don't think it make sense to mention any API. This is for mono runtime so what block anyone to run this with their own set of libraries? It should explain what it does and how to use it